src/app/shared/markdown-wrapper/markdown-wrapper.service.ts

```diff
@@ -77,18 +78,22 @@ export class MarkdownWrapperService {
     'onunload',
   ];
 
-  constructor(private markdownService: MarkdownService) {}
+  constructor() {}
 
   /**
    * Compiles markdown into HTML with custom options.
    * @param data
-   * @param baseUrl A base url used as a prefix for relative links
+   * @param markdownBaseUrl A base url used as a prefix for relative links
    * @returns {string} HTML string
    */
-  customCompile(data, baseUrl): string {
-    const parseOptions = { markedOptions: { baseUrl: baseUrl } };
```

Comment:
In this service, I switched us from using `ngx-markdown`'s `MarkdownService` to using `marked` in order to parse markdown to HTML. Under the hood, `ngx-markdown` uses `marked` to do this anyway, so we shouldn't see a difference in the UI.

The reason for this change is that `baseUrl` is no longer a `markedOption` property because it was removed from the `marked` package in an updated version. Configuring the base url is now done through an extension package called `marked-base-url`. It is considerably easier to directly use the `marked` package to do this instead of the `ngx-markdown` package hence the switch.